lib/knapsack_pro/tracker.rb

```diff
@@ -110,6 +110,7 @@ def save_prerun_tests_report(hash)
     end
 
     def read_prerun_tests_report
+      raise "Report #{prerun_tests_report_path} doest not exist on the disk. Most likely, it was removed accidentally. Please report the bug to the support team." unless File.exists?(prerun_tests_report_path)
```

Comment:
Maybe we can add a link here, or at least specify that we mean the Knapsack support team?